### PR TITLE
Updating Dockerfiles From Docker Directory and get_started.md to Reach Last Stable Version of Python, PyTorch and MMCV

### PR DESCRIPTION
**Pull request by Nourollah**
Motivation
My master thesis project was developed using MMDetection and MMSegmentation and I'm very interested in OpenMMLab projects. I particularly like the latest version of the technology. For 3 months, PyTorch did not support Python3.10. However, version 1.11.0 of PyTorch adds Python3.10 support.

Modification
The Dockerfile in the docker directory has been updated to reach the latest versions of MMCV and PyTorch. In dockerfiles, some commands were added to gain a smaller docker image. In documents, 'get_started.md' contains updates to the last version of Python, MMCV, and PyTorch with respect to BC.

BC-breaking (Optional)
No, this version of MMCV and PyTorch have fully backward compatibility. This is good for moving forward by safe backward compatibility.

Use cases (Optional)
Nothing changed in providing new use cases.

### CODE REVIEW

**Review by MeowZheng — APPROVED** [review]
Mang thanks for your contribution, and it is very helpful.

File: docs/zh_cn/get_started.md [hunk 1]
```diff
@@ -155,11 +155,11 @@ pip install -e .  # 或者 "python setup.py develop"
...
 
 ```shell
-conda create -n open-mmlab python=3.7 -y
+conda create -n open-mmlab python=3.10 -y
 conda activate open-mmlab
 
-conda install pytorch=1.6.0 torchvision cudatoolkit=10.1 -c pytorch
-pip install mmcv-full==latest+torch1.5.0+cu101 -f https://download.openmmlab.com/mmcv/dist/index.html
+conda install pytorch=1.11.0 torchvision cudatoolkit=11.3 -c pytorch
+pip install mmcv-full==latest+torch1.11.0+cu113 -f https://download.openmmlab.com/mmcv/dist/index.html
```
Comment (MeowZheng) [inline docs/zh_cn/get_started.md:162]:
```suggestion
pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu113/torch1.11.0/index.html
```

Comment (MeowZheng) [inline docs/zh_cn/get_started.md:162]:
```suggestion
pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu113/torch1.11.0/index.html
```

**Review by MengzhangLI — APPROVED** [review]

**Comment by Nourollah** [comment]
> Thanks for your excellent contribution especially modification in chinese doc.

Thanks for reviewing my contribution.

**Comment by jason102811** [comment]
nourollah，您好！您在MMSeg项目中给我们提的PR非常重要，感谢您付出私人时间帮助改进开源项目，相信很多开发者会从你的PR中受益。
我们非常期待与您继续合作，OpenMMLab专门成立了贡献者组织MMSIG，为贡献者们提供开源证书、荣誉体系和专享好礼，可通过添加微信：openmmlabwx 联系我们（请备注mmsig+GitHub id），由衷希望您能加入！
Dear nourollah,
First of all, we want to express our gratitude for your significant PR in the MMSeg project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.
We are looking forward to continuing our collaboration with you. OpenMMLab has established a special contributors' organization called MMSIG, which provides contributors with open-source certificates, a recognition system, and exclusive rewards. You can contact us by adding our WeChat（if you have WeChat): openmmlabwx, or join in our discord： https://discord.gg/qH9fysxPDW. We sincerely hope you will join us!
Best regards！ @Nourollah

**Pull request merged**